cli/options.json

```diff
@@ -260,7 +260,7 @@
   },
   "lowMemoryLimit": {
     "category": "Features",
-    "description": "Enforces very low (<64k) memory constraints.",
+    "description": "Enforces memory constraints, memory usage will not exceed the given value.",
```

Comment:
```suggestion
    "description": "Enforces memory constraints; memory usage will not exceed the given address.",
```

idk; @dcodeIO wdyt?

Reply:
Primary challenge here is line length :)